LendioDevs/flutter_zendesk_messaging
# Upgrade package

To test, change dependency to the following:
```
zendesk_messaging:
    git:
      url: https://github.com/LendioDevs/flutter_zendesk_messaging
      ref: upgrade
```

and run the following commands to clean cache:
```
flutter pub cache clean
flutter pub get
```

Then, open the chat to Android and iOS, it should work.
